examples/lock-app/silabs/include/LockManager.h

```diff
@@ -199,10 +199,12 @@ class LockManager
 private:
     struct UnlatchContext
     {
+        static constexpr uint8_t kMaxPinLenght = UINT8_MAX;
+        uint8_t mPinBuffer[kMaxPinLenght];
```

Comment:
why the switch to an array instead of a bytespan?

Reply:
Bytespan doesn't own its memory.

Reply:
True - i forgot about that